client/rest-high-level/src/test/java/org/elasticsearch/client/documentation/IndicesClientDocumentationIT.java

```diff
@@ -2128,4 +2133,108 @@ public void onFailure(Exception e) {
 
         assertTrue(latch.await(30L, TimeUnit.SECONDS));
     }
+
+    public void testPutStoredScript() throws Exception {
+        RestHighLevelClient client = highLevelClient();
+
+        {
+            createIndex("index1", Settings.EMPTY);
+        }
+
+        {
+            // tag::put-stored-script-request
+            PutStoredScriptRequest request = new PutStoredScriptRequest();
+            request.id("id"); // <1>
+            request.content(new BytesArray(
+                "{\n" +
+                    "\"script\": {\n" +
+                    "\"lang\": \"painless\",\n" +
+                    "\"source\": \"Math.log(_score * 2) + params.multiplier\"" +
+                    "}\n" +
+                    "}\n"
+            ), XContentType.JSON); // <2>
+            // end::put-stored-script-request
+        }
+
+        {
+            PutStoredScriptRequest request = new PutStoredScriptRequest();
```

Comment:
same - pls move it to `PutStoredScriptRequest.toXContent(XContentBuilder builder, Params params)`

Reply:
@vladimirdolzhenko Thanks for your advice. But I am curious about that is it necessary to move the code to `PutStoredScriptRequest.toXContent(XContentBuilder builder, Params params)`. I found there are lots of test code written like this. 

And as far as I know, `toXContent` will build a `XContentBuilder` object based on the content of a request, and it will be used in `RequestConverters`. So I am not really sure how to put this into `PutStoredScriptRequest.toXContent`. If you have any idea how to do that, please let me know and I will fix this as soon as possible. Thanks!

Reply:
Sorry, my fault - I thought there is a method to set `source` directly - you've already added `PutStoredScriptRequest.toXContent` that creates nested `script` object with source itself - so that part is good.